src/govuk/components/exit-this-page/_index.scss

```diff
@@ -44,12 +43,27 @@
     width: .5em;
     height: .5em;
     margin: 0 .125em;
-    border: 2px solid govuk-colour("white");
+    border-width: 2px;
+    border-style: solid;
     border-radius: 50%;
+
+    @include govuk-not-ie8 {
+      border-color: currentcolor;
+    }
+
+    @include govuk-if-ie8 {
+      border-color: govuk-colour("white");
+    }
   }
 
   .govuk-exit-this-page__indicator-light--on {
-    background-color: govuk-colour("white");
+    @include govuk-not-ie8 {
+      background-color: currentcolor;
```

Comment:
Because backgrounds get removed in forced colors mode, the fill still doesn't show up even though we're using `currentcolor`.

It might be worth using a similar technique we use for radio buttons, where we make the entire thing out of border?

Reply:
Good point! Will look at reworking how we light these up.

Reply:
Actually, I'm going to progress this separately, as it isn't a new issue introduced by this PR and I'd rather not block the fixes with unrelated refactoring.